src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/caseupdate/services/CaseUpdaterTest.java

```diff
@@ -121,13 +122,46 @@ public void should_update_case_data_with_exception_record() {
             null
         );
 
+        CaseUpdateDetails caseUpdateDetails = new CaseUpdateDetails(
```

Comment:
This method is ~200 lines long. We should extract things like this (and ideally other similar long bits) into a separate method.

Reply:
ok

Reply:
There is another task covering this: https://tools.hmcts.net/jira/browse/BPS-1330